lib/src/internal/http_endpoints.dart

```diff
@@ -620,12 +653,19 @@ class HttpEndpoints implements IHttpEndpoints {
   }
 
   @override
-  Future<void> modifyCurrentMember(Snowflake guildId, {String? nick}) async =>
-      executeSafe(BasicRequest("/guilds/$guildId/members/@me/nick", method: "PATCH", body: {if (nick != null) "nick": nick}));
+  Future<void> modifyCurrentMember(Snowflake guildId, {String? nick}) async => executeSafe(BasicRequest(
+      HttpRoute()
```

Comment:
Cna you provide on HttpRoute class small documentation/explanation how to use that syntax. It could be kinda confusing for new contributors or for use after time